service/src/main/java/com/codeforcommunity/processor/authenticated/ProtectedSchoolProcessorImpl.java

```diff
@@ -872,4 +898,14 @@ private SchoolsRecord queryForSchool(int schoolId) {
   private boolean isShipmentYearInvalid(Integer year) {
     return year <= 999 || year >= 10000;
   }
+
+  private String getUserName(int userId) {
+    UsersRecord userRecord = db.selectFrom(USERS).where(USERS.ID.eq(userId)).fetchOne();
+    return userRecord.getFirstName() + " " + userRecord.getLastName();
```

Comment:
Let's add a `null` check here for `userRecord` (probably throw some sort of exception and log an error since this should never happen?)